src/ripple/app/misc/impl/Transaction.cpp

```diff
@@ -165,13 +165,19 @@ Transaction::load(
 
 // options 1 to include the date of the transaction
 Json::Value
-Transaction::getJson(JsonOptions options, bool binary) const
+Transaction::getJson(
+    JsonOptions options,
+    bool binary,
+    bool showInLedger,
+    std::optional<std::reference_wrapper<std::string>> hash) const
 {
-    Json::Value ret(mTransaction->getJson(JsonOptions::none, binary));
+    Json::Value ret(mTransaction->getJson(JsonOptions::none, binary, hash));
 
     if (mInLedger)
```

Comment:
Since `inLedger` field is being deprecated, it would be nice to rename the `mInLedger` data member into `mLedgerIndex` or something like that. It would help with the readability of the code.

Reply:
Thanks, will do that. In general I do not like mixing refactoring with functional changes, but here it is a clear readability improvement with very small added churn.